pkg/cosign/kubernetes/api/v1alpha1/imagepolicy_types.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// ClusterImagePolicySpec defines the desired state of ImagePolicy
+type ClusterImagePolicySpec struct {
```

Comment:
To reiterate my request on the issue:
> I think we should probably start a doc enumerating key scenarios (perhaps ... can help provide a "voice of the user" to keep us honest?), and then sketch a more formal API spec where we spell out the fields and semantics. I'd be happy to collaborate if we want to get a group of interested folks spun up around this?

Reply:
Sorry, relatively new here! so I have a few questions. 
Do you have a drive where we can create/save a doc to detail this? I can start the doc. Whats the process to form an interest group? Do we just open this conversation up in the community meeting?

Reply:
I think some sort of limited run API WG/task-force might be better than hashing it out in the bigger community meeting.  Maybe we can ask / socialize this at the next community call?  (cc @lukehinds @dlorenc )

Reply:
add it to the agenda @mattmoor , sounds like a good idea!

Reply:
@mattmoor et al,

had a chat with the TAC and recommend:

1. Set a date in the 2nd half of next week (or later) to meet on hangouts (to give enough time to invite others on next weeks community call)
2. Add sigstore-dev@googlegroups.com to the invite
3. You can repeat it for as long as you like.
4. Might be worth recording as well, I can add to the sigstore community YouTube channel